tests/unit/test_cli.py

```diff
@@ -581,12 +587,41 @@ def test_migrate_locations_azure(ws):
         azure_subscription_id='test',
         azure_resources=azurerm,
     )
+
     migrate_locations(ws, ctx=ctx)
+
     ws.external_locations.list.assert_called()
     azurerm.storage_accounts.assert_called()
 
 
-def test_migrate_locations_aws(ws, caplog):
+@pytest.mark.xfail(reason="Currently not supported in unit tests see TODO")
```

Comment:
Linking this to track another example belonging to https://github.com/databrickslabs/ucx/issues/2507